frontend/src/Editor/Components/Map.jsx

```diff
@@ -110,6 +192,7 @@ export const Map = function Map({
           mapContainerStyle={containerStyle}
           zoom={12}
           options={{
+            styles: localStorage.getItem('darkMode') === 'true' ? darkModeStyles : '',
```

Comment:
Every widget will get `darkMode` as a parameter, we don't need to fetch from local storage. Can you please check `frontend/src/Editor/Components/Chart.jsx` and implement this logic in a similar way?

Reply:
Using  Prop